src/components/Feedback.vue

```
@@ -91,19 +118,23 @@
     })
   })
 
+  if (response.ok) {
+    toastBootstrap.show()
```

ESLint

[error] 'toastBootstrap' is not defined.

(no-undef)